filter/filter.go

```diff
@@ -96,6 +99,7 @@ func (c *Client) GetOutput(ctx context.Context, serviceAuthToken, downloadServic
 		return m, err
 	}
 
+	common.AddUserHeader(req, userAuthToken)
```

Comment:
I think this will be setting the wrong header for this value. `AddUserHeader` adds a `User-Identity` header, whereas we'd want to set the `X-Florence-Token` header.

Example:
https://github.com/ONSdigital/dp-api-clients-go/blob/master/dataset/dataset.go#L505